packages/labs/compiler/src/lib/template-transform.ts

```diff
@@ -139,6 +147,28 @@ class CompiledTemplatePass {
     return findTemplates(sourceFile);
   }
 
+  /**
+   * Returns true if passed a node that marks the file for compilation.
+   * Currently files are marked for compilation if an importClause is detected
+   * that contains an `html` import from `lit` or `lit-html`.
+   */
+  private shouldMarkFileForCompilation<T extends ts.Node>(node: T): boolean {
+    if (
+      ts.isImportDeclaration(node) &&
+      ts.isStringLiteral(node.moduleSpecifier) &&
+      (node.moduleSpecifier.text === 'lit' ||
+        node.moduleSpecifier.text === 'lit-html') &&
+      node.importClause?.namedBindings != null &&
+      ts.isNamedImports(node.importClause.namedBindings)
+    ) {
+      const namedBindings = node.importClause.namedBindings;
+      return namedBindings.elements.some(
+        (imp) => ts.isIdentifier(imp.name) && imp.name.text === 'html'
+      );
+    }
+    return false;
+  }
```

Comment:
Can we resolve the reference for the template tag function instead? I'm concerned this approach would miscompile:

```ts
import {html} from 'lit';
import * as static from 'lit/static.js';

function foo() {
  const html = static.html;
  return html``;
}
```

Reply:
Also, if we go with the currently implemented approach, does it properly handle code like this:

```ts
import {svg as html} from 'lit';
```

Reply:
Yes – those cases will miscompile. I can add resolution.

Reply:
Also handle:
```
import * as lit from 'lit';

lit.html`rar`;
```